model/config.go

```diff
@@ -481,6 +507,71 @@ func (store *istioConfigStore) Policy(instances []*ServiceInstance, destination
 	return out
 }
 
+// DefaultProxyConfig for individual proxies
+func DefaultProxyConfig() proxyconfig.ProxyConfig {
+	return proxyconfig.ProxyConfig{
+		ConfigPath:             "/etc/istio/proxy",
+		BinaryPath:             "/usr/local/bin/envoy",
+		ServiceCluster:         "istio-proxy",
+		DrainDuration:          ptypes.DurationProto(2 * time.Second),
+		ParentShutdownDuration: ptypes.DurationProto(3 * time.Second),
+		DiscoveryAddress:       "istio-pilot:8080",
+		DiscoveryRefreshDelay:  ptypes.DurationProto(1 * time.Second),
```

Comment:
Can we change the default in the next PR please? to 10s or something

Reply:
Ok